proxy/vm.go

```diff
@@ -16,14 +16,12 @@ package main
 
 import (
 	"encoding/binary"
-	"errors"
 	"fmt"
-	"math"
 	"net"
 	"os"
 	"sync"
 
-	hyper "github.com/hyperhq/runv/hyperstart/api/json"
+	"github.com/sameo/virtcontainers/hyperstart"
```

Comment:
PR#422 allows the proxy to rely on the hyperstart package from virtcontainers. This means that we use the same codebase to discuss with hyperstart for both CC2.1 and CC3.0 projects. The link you mentioned " https://github.com/clearlinux/hyperstart" corresponds to the hyperstart binary which is running inside the VM itself.